buildSrc/src/main/kotlin/ComposePlatforms.kt

```diff
@@ -79,6 +79,19 @@ enum class ComposePlatforms(vararg val alternativeNames: String) {
             ComposePlatforms.MingwX64,
         )
 
+        // Let's not publish those for some libs until we have a reason to do so
+        // For example: lifecycle
+        val SKIP_EXTRA_TARGETS = EnumSet.of(
```

Comment:
The difference between `NO_SKIKO` is only in Linux targets, and it isn't obvious without reading the code.

I would suggest remove it and either:
1. Substract Linux in `PublishedLifecyclePlatforms` (we already do that for Arm)
2. (I prefer this) Just publish Linux - less code branching (easier to understand), no extra support (we have other native targets), aligned with other libraries.

Reply:
Could you also fix this?

Reply:
Sure. The idea was to save the CI resources since we regularly face some issues with it, and we don't know yet a use case for linuxArm64 libs.

But I enabled them (in 3 libraries: kruth, lifecycle-common, lifecycle-runtime). Can be disabled later if needed.